### DISCUSSION TITLE
How many cameras can a RPi4 handle without a coral?

### DISCUSSION THREAD

**Discussion by srlowe** [blakeblackshear/frigate#2143] [Ask A Question]
Since it's impossible to get a USB coral TPU at the moment, I'd like to ask how many cameras a Raspberry Pi4 (4GB RAM) could realistically handle without it?  I've read the docs, and it say's "a small number" but I'm unclear on what that means exactly.  Could it handle 4 cameras?

EDIT: The my use case would involve very little activity (normally), and for framerate and resolution - I suppose the lowest that would still be useful as a security setup.

Thanks.

**Comment by blakeblackshear** [comment]
It depends on resolutions, framerates, and how much activity you have on the cameras. Without a Coral, an RPi4 is going to run a very limited number of cameras.   Even 1 may consume all it's resources. Maybe some others will chime in with their experience.

**Reply by srlowe** [reply]
I see, thank you. 

Activity would be very low.  I'm flexible on framerate and resolution - I suppose whatever minimums are still usable as security cameras.
